docs/data-download.md

```diff
@@ -30,7 +30,7 @@ usage: freqtrade download-data [-h] [-v] [--logfile FILE] [-V] [-c PATH]
 optional arguments:
   -h, --help            show this help message and exit
   -p PAIRS [PAIRS ...], --pairs PAIRS [PAIRS ...]
-                        Show profits for only these pairs. Pairs are space-
+                        Only download data for these pairs. Pairs are space-
```

Comment:
that won't change the output of the help command though ... and that is ... difficult to make it work for all - as the same argument is used by plot commands, as well as download data and list-data (and convert_data)